lib/iris/tests/results/netcdf/netcdf_save_gridmapmulti.cdl

```diff
@@ -15,12 +15,12 @@ variables:
 		latitude_longitude:earth_radius = 6371229. ;
 	int64 longitude(longitude) ;
 		longitude:axis = "X" ;
-		longitude:units = "degrees_east" ;
+		longitude:units = "1" ;
```

Comment:
which test is it?
may I suggest that perhaps a more useful test would be where they are created with a units of 'degrees'?